README.md

```diff
@@ -119,6 +115,14 @@ Docker and Docker Compose are the best way of rendering the site locally on a Ma
 
 You can now follow the rest of the instructions to render the site locally.  The first time you serve the site locally you'll be asked to allow ruby to listen for network connections, you should agree to this.
 
+### Docker setup
+
+Whilst there is a docker-compose file to render the site locally, it is largely unsupported and frequently breaks.  The instructions below may work for you, but we are considering removing this completely.
```

Comment:
> we are considering removing this completely.

Reword something like this, perhaps?

"We are considering removing this completely, but if anybody can get this to work consistently, we would be happy to accept a Pull Request.

Reply:
wording updated